constellations/orion/src/repos/achieve_repo.go

```diff
@@ -170,8 +174,9 @@ func (ar *achieveRepo) Insert(achieve domains.Achieve) error {
 		"created_at, " +
 		"updated_at, " +
 		"year, " +
-		"message" +
-		") VALUES (?, ?, ?, ?)"
+		"message, " +
+		"`order` " +
```

Comment:
why does order here have `?

Reply:
apparently "order" is a keyword in mysql, so I had to add backticks to make it work

Reply:
oh interesting.